content/modules/r/pages/subset.adoc

```diff
@@ -0,0 +1,165 @@
+= `subset`
+
+`subset`  is a function that helps you take subsets of data. By default, `subset` removes NA rows, so use with care. `subset` does not perform any operation that can't be accomplished by indexing, but can sometimes be easier to read.
+
+Where we would normally write something like:
+
+[source, R]
+----
+grades[grades$year=="junior" | grades$sex=="M",]$grade
+----
+
+[source, R]
+----
+[1] 100  75  74  69  88  99  90  92
+----
+
+What we can instead do:
+
+[source, R]
+----
+subset(grades, year=="junior" | sex=="M", select=grade)
+----
+
+[source, R]
+----
+   grade
+1    100
+3     75
+4     74
+6     69
+7     88
+8     99
+9     90
+10    92
+----
+
+But be careful, if we replace a grade with an NA, it will be removed by subset:
+
+[source, R]
+----
+grades$sex[8] <- NA
+subset(grades, year=="junior" | sex=="M", select=grade)
+----
+
+[source, R]
+----
+   grade
+1    100
+3     75
+4     74
+6     69
+7     88
+9     90
+10    92
+----
+
+Whereas indexing will not unless you specify to:
+
+[source, R]
+----
+grades[grades$year=="junior" | grades$sex=="M",]$grade
+----
+
+[source, R]
+----
+[1] 100  75  74  69  88  NA  90  92
+----
+
+== Examples
+
+=== How can I easily make a subset of the 8451 data, using only 1 line of R, with the `subset` function?
+
+.Click to see solution
+[%collapsible]
+====
+https://cdnapisec.kaltura.com/html5/html5lib/v2.79.1/mwEmbedFrame.php/p/983291/uiconf_id/29134031/entry_id/1_2gzdeg6h?wid=_983291&iframeembed=true&playerId=kaltura_player&entry_id=1_2gzdeg6h&flashvars%5BstreamerType%5D=auto&flashvars%5BlocalizationCode%5D=en&flashvars%5BleadWithHTML5%5D=true&flashvars%5BsideBarContainer.plugin%5D=true&flashvars%5BsideBarContainer.position%5D=left&flashvars%5BsideBarContainer.clickToClose%5D=true&flashvars%5Bchapters.plugin%5D=true&flashvars%5Bchapters.layout%5D=vertical&flashvars%5Bchapters.thumbnailRotator%5D=false&flashvars%5BstreamSelector.plugin%5D=true&flashvars%5BEmbedPlayer.SpinnerTarget%5D=videoHolder&flashvars%5BdualScreen.plugin%5D=true&flashvars%5BKaltura.addCrossoriginToIframe%5D=true&&wid=1_sqr78rpp[Video Explanation]
+
+In the 84.51 data set:
+
+[source, R]
+----
+myDF <- read.csv("/class/datamine/data/8451/The_Complete_Journey_2_Master/5000_transactions.csv")
+----
+
+We recall that these are the variables:
+
+[source, R]
+----
+head(myDF)
+----
+
+and there are 10625553 rows and 9 columns
+
+[source, R]
+----
+dim(myDF)
+----
+
+We can use the `subset` command to focus on only the purchases from the `CENTRAL` store region, in the `YEAR 2016`. We can also pick which variables that we want to have in this new data frame.
+
+Please note: We do not need to specify `myDF` on each variable, because the `subset` function will keep track of this for us. The `subset` function knows which data set that we are working with, because we specify it as the first parameter in the `subset` function.
+
```

Comment:
Add the explanation of how the `subset` function is set up. 

`subset(x, subset, select, drop = FALSE, …)`